website/docs/docs/build/ref-metrics-in-filters.md

```diff
@@ -0,0 +1,164 @@
+
+- Create two semantic models: `model_runs` and `accounts`.
+- Create a `measure` and `metric` to count data model runs, and another measure to count users.
+- Specify the foreign entity `account` in the `model_runs` semantic model.
+- Then create the `Activated Accounts` metric by filtering accounts that have more than five data model runs.
+
+```yaml
+semantic_models:
+  - name: model_runs
+    ... # Placeholder for other configurations
+    entities:
+      - name: model_run
+        type: primary
+      - name: account
+        type: foreign
+    measures:
+      - name: data_model_runs
+        agg: sum
+        expr: 1
+        create_metric: true
+
+  - name: accounts
+    ... # Placeholder for other configurations
+    entities:
+      - name: account
+        type: primary
+    measures:
+      - name: accounts
+        agg: sum
+        expr: 1
+metrics:
+  - name: activated_accounts
+    label: Activated Accounts
+    type: simple
+    type_params:
+      measure: accounts
+    filter: |
+      {{ Metric('data_model_runs', group_by=['account']) }} > 5
+```
+
+Let’s break down the SQL the system generates based on the metric definition when you run `dbt sl query --metrics activated_accounts` from the command line interface:
+
+- The filter `{{ Metric('data_model_runs', group_by=['account']) }}` generates SQL similar to the `data_models_per_user` sub-query shown earlier:
+
+	```sql
+	select
+		sum(1) as data_model_runs,
+		account
+	from 
+		data_model_runs
+	group by
+		account
+	```
+
+- MetricFlow joins this query to the query generated by the `accounts` measure on the group by elements and applies the filter conditions:
+
+	```sql
+	select
+		sum(1) as activated_accounts
+	from accounts
+	left join (
+	select
+		sum(1) as data_model_runs, 
+		account
+	from data_model_runs
+	group by 
+		account
+	) as subq
+	on accounts.account = subq.account
+	where data_model_runs > 5
+	```
+
+The intermediate tables used to create this metric are: Accounts with the `data_model_runs` dimension
+
+| account | data_model runs |
+| --- | --- |
+| 1 | 4 |
+| 2 | 7 |
+| 3 | 9 |
+| 4 | 1 |
+
+Then, filter this table to accounts with more than 5 data model runs and count the number of accounts that meet this criteria:
+
+| activated_accounts |
+| --- |
+| 2 |
+
+## Considerations when using metrics in filter
+
+- When using a metric filter, ensure the sub-query can join to the outer query without fanning out the result ( unexpectedly increasing the number of rows). 
+  - The example that filters the accounts measure using `{{ Metric('data_model_runs', group_by=['account']) }}` is valid because it aggregates the model runs to the account level.
+  - However, filtering by `{{ Metric('data_model_runs', group_by=['model']) }}` isn't valid due to a one-to-many relationship between accounts and model runs, leading to duplicate data.
```

Comment:
I would update this to say `However, filtering the 'accounts' measure by...`
Currently it's hard to remember what we're filtering and that's crucial to understanding the one-to-many relationship mentioned here.